blueprints/codes/latex_formula.py

```diff
@@ -57,7 +54,7 @@ def short(self) -> str:
             Return symbol = result
 
         """
-        return f"{self.return_symbol} {self.comparison_operator_label} {self.result}"
+        return f"{self.return_symbol} {self.comparison_operator_label} {self.result}{self.unit}"
```

Comment:
please add a space between result and unit

Reply:
Done